src/Processors/DataProcessor.php

```diff
@@ -125,13 +126,28 @@ public function process($object = false)
     protected function addColumns($data, $row)
     {
         foreach ($this->appendColumns as $key => $value) {
-            $value['content'] = Helper::compileContent($value['content'], $data, $row);
+            $value['content'] = Helper::compileContent($value['content'], $data, $row, $this->shouldEscapeColumn($key));
```

Comment:
I think the escape flag here is not needed since escaping will be done on `escapeRow` process.